tools/jslint/jslint.js

```diff
@@ -607,7 +607,9 @@ var JSLINT = (function () {
             use_object: "Use the object literal notation {}.",
             use_or: "Use the || operator.",
             use_param: "Use a named parameter.",
-            used_before_a: "'{a}' was used before it was defined.",
+            used_before_a: "'{a}' was used before it was defined. (If you have" +
+                " added a new global, you need to add it to the whitelist in" +
+                " tools/jslist/check-all.js)",
```

Comment:
This is a great improvement, but it should probably be a separate commit from the new feature.  Probably a reasonable  approach would be to create a new PR that's just this change, we can merge that, and then when we rebase this change, git will automatically drop this hunk from this commit.